docs/src/pages/demos/text-fields/InputWithIcon.js

```diff
@@ -43,6 +43,7 @@ function InputWithIcon(props) {
           ),
         }}
       />
+
```

Comment:
Now, I understand that the bank lines were added on purpose. Why? The TypeScript demos don't them. From my perspective, I think that adding space is arbitrary. To remove this layer of thinking, I think that it's simpler to never add some.

Reply:
@oliviertassinari They are not added on purpose. They are a transpilation artifact. When babel strips the types it removes some characters and it tries to fill that void with blanklines.